CI build job humble/testing failed!

An automated scheduled build failed on `master`: https://github.com/ros-controls/ros2_control_ci/actions/runs/24871943359

```
 /home/runner/work/ros2_control_ci/ros2_control_ci/.work/target_ws/src/ros-controls/topic_based_hardware_interfaces/cm_topic_hardware_component/test/cm_topic_hardware_component_test.cpp: In constructor ‘TestableResourceManager::TestableResourceManager(const string&, bool, bool)’:
  /home/runner/work/ros2_control_ci/ros2_control_ci/.work/target_ws/src/ros-controls/topic_based_hardware_interfaces/cm_topic_hardware_component/test/cm_topic_hardware_component_test.cpp:156:82: error: no matching function for call to ‘hardware_interface::ResourceManager::ResourceManager(const string&, bool&, bool&)’
    156 |     : hardware_interface::ResourceManager(urdf, validate_interfaces, activate_all)
        |                                                                                  ^

```

--- a/mujoco_ros2_control_demos/package.xml
+++ b/mujoco_ros2_control_demos/package.xml
@@ -20,7 +20,7 @@
   <exec_depend>joint_state_broadcaster</exec_depend>
   <exec_depend>mujoco_ros2_control</exec_depend>
   <exec_depend>mujoco_ros2_control_msgs</exec_depend>
-  <exec_depend>position_controllers</exec_depend>
+  <exec_depend>forward_command_controller</exec_depend>
   <exec_depend>robot_state_publisher</exec_depend>
   <exec_depend>rviz2</exec_depend>
   <exec_depend>xacro</exec_depend>

--- a/mujoco_ros2_control_tests/package.xml
+++ b/mujoco_ros2_control_tests/package.xml
@@ -24,7 +24,7 @@
   <test_depend>mujoco_ros2_control_demos</test_depend>
   <test_depend>mujoco_ros2_control_msgs</test_depend>
   <test_depend>pluginlib</test_depend>
-  <test_depend>position_controllers</test_depend>
+  <test_depend>forward_command_controller</test_depend>
   <test_depend>rclcpp</test_depend>
   <test_depend>robot_state_publisher</test_depend>
   <test_depend>rviz2</test_depend>

